File: app/src/main/java/com/example/University_blind_dating/ui/chatting/ChattingFragment.java
package com.example.University_blind_dating.ui.chatting;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.lifecycle.Observer;
import androidx.lifecycle.ViewModelProviders;

import com.example.University_blind_dating.R;

public class ChattingFragment extends Fragment {

    private ChattingViewModel chattingViewModel;

    public View onCreateView(@NonNull LayoutInflater inflater,
                             ViewGroup container, Bundle savedInstanceState) {
        chattingViewModel = ViewModelProviders.of(this).get(ChattingViewModel.class);
        View root = inflater.inflate(R.layout.fragment_chatting, container, false);

        return root;
    }
}

File: app/src/main/java/com/example/myapplication/ui/chatting/ChattingFragment.kt
package com.example.myapplication.ui.chatting

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.fragment.app.Fragment
import androidx.lifecycle.Observer
import androidx.lifecycle.ViewModelProviders
import com.example.myapplication.R

class ChattingFragment : Fragment() {

    private lateinit var chattingViewModel: ChattingViewModel

    override fun onCreateView(
            inflater: LayoutInflater,
            container: ViewGroup?,
            savedInstanceState: Bundle?
    ): View? {
        chattingViewModel =
                ViewModelProviders.of(this).get(ChattingViewModel::class.java)
        val root = inflater.inflate(R.layout.fragment_dashboard, container, false)
        val textView: TextView = root.findViewById(R.id.text_dashboard)
        chattingViewModel.text.observe(viewLifecycleOwner, Observer {
            textView.text = it
        })
        return root
    }
}